spec/models/form_profile_spec.rb

```diff
@@ -866,6 +866,48 @@
     }
   end
 
+  let(:v21_22_expected_) do
+    {
+      'personalInformation' => {
+        'first' => user.first_name&.capitalize,
+        'middle' => user.middle_name&.capitalize,
+        'last' => user.last_name&.capitalize,
+        'suffix' => user.suffix,
+        'preferredName' => 'SAM',
+        'dateOfBirth' => user.birth_date,
+        'socialSecurityNumber' => user.ssn,
+        'serviceNumber' => '123455678'
+      },
+      'contactInformation' => {
+        'email' => user.pciu_email,
+        'phone' => us_phone,
+        'address' => address
+      },
+      'veteranServiceInformation' => veteran_service_information
+    }
+  end
+
+  let(:v21_22a_expected_) do
```

Comment:
Delete the trailing `_` in `v21_22a_expected_`.

Also, where is this being used?